lib/src/collection/lru_map.dart

```diff
@@ -0,0 +1,286 @@
+part of quiver.collection;
+
+/**
+ * An implementation of a [Map] which has a maximum size and uses a Least
+ * Recently Used algorithm to remove items from the [Map] when the [maximumSize]
+ * is reached and new items are added.
```

Comment:
Should we mention that using `keys` and `values` does not affect keys' "used" ordering?